howto/cluster/configure-nodes.md

```diff
@@ -0,0 +1,54 @@
+Your cluster might contain nodes with different resources and different capacity. Therefore, each node can be configured separately.
+
+## Show node configuration
+
+To display the current node configuration for a node (for example, `lxd0`), enter the following command:
+
+    amc node show <node>
+
+This command will return output similar to the following:
+
+```bash
+name: lxd0
+status: online
+disk:
+    size: 100GB
+network:
+    address: 10.119.216.34
+    bridge-mtu: 1500
+config:
+    public-address: 10.119.216.34
+    use-port-forwarding: true
+    cpu-cores: 8
+    cpu-allocation-rate: 4
+    memory: 16GB
+    memory-allocation-rate: 2
+    gpu-slots: 10
+    gpu-encoder-slots: 0
+    tags: []
+```
+
+<a name="configure-allocation-rates"></a>
+## Configure allocation rates
+
+AMS allows over-committing available resources on a node. This mechanism improves resource usage, because usually, containers don't use 100% of their dedicated resources all of the time.
+
+By default, AMS uses a CPU allocation rate of `4` and a memory allocation rate of `2`. See [Over-committing](tbd#overcommitting) for more information.
+
+You can configure the allocation rates with the `cpu-allocation-rate` and `cpu-memory-allocation-rate` configuration items.
+
+Use the following commands to set the allocation rates on a node (for example, `lxd0`):
+
+    amc node set <node> cpu-allocation-rate <value>
+    amc node set <node> memory-allocation-rate <value>
+
+<a name="configure-gpu-slots"></a>
+## Configure GPU slots
+
+GPU slots are used to share GPUs amongst containers. See [About GPU support](https://discourse.ubuntu.com/t/gpu-support/17768) and [GPU slots](tbd#gpu-slots) for more information.
+
+Each GPU-equipped cluster node must be configured with a number of GPU slots. (Nodes without GPU are configured with 0 GPU slots.)
```

Comment:
Is that correct? Or would nodes without GPU just not have gpu-slots set?

Reply:
AMS will assign a default value for gpu-slots and gpu-encoder-slots so users don't have to manually configure them unless there is a need. See the values in the comment below.

would nodes without GPU just not have gpu-slots set
A: no, it wouldn't. The value `0` will be assigned for both slots.